crates/extensions/c8y_mapper_ext/src/supported_operations.rs

```diff
@@ -0,0 +1,501 @@
+//! [Supported Operations API]
```

Comment:
I just find the placement of this module a bit weird, as it is placed directly under `operations` whereas `operations.rs` is directly placed under `supported_operations`. May be you want to move it into a `mod.rs` under `supported_operations` or place both directly under `operations`?

Reply:
This style of module placement was supported for some time, and I think it's better to use than the older style for 2 reasons:

- if you split a submodule off a bigger module, you don't have to move the other file and rename it to `mod.rs`, which makes the diffs a bit smaller (bigger pro IMO)
- the file name is still a module name, whereas with `mod.rs` for the module name you have to look at the name of the parent directory, but as the editor tab contains only a file name, you can end up with many `mod.rs` files, so it's a bit harder to keep track of (actually in vscode when two files have the same name then it will display a parent in the tab for context)

But as we're already using the older style (we have 38 `mod.rs` files) I will use this style for consistency.